<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>在线个人财务规划应用</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="tool" native="true">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#tool {
    background-color: rgb(200, 200, 200);
}

/* 按钮样式 */
QPushButton {
    width: 150px;
    height: 55px;
    background-color: rgb(200, 200, 200); /* 背景色为浅灰色 */
    border: none;
    color: white; /* 文本颜色为白色 */
    font-size: 16px; /* 字体大小 */
    font-family: Arial, sans-serif; /* 字体选择，根据需要调整 */
    font-weight: bold; /* 字体粗细，可以根据需要调整为 normal 或 lighter */
}

/* 鼠标悬停状态 */
QPushButton:hover {
    background-color: rgb(150, 150, 150); /* 悬停时的背景色 */
    color: white; /* 悬停时的文本颜色 */
}

/* 按钮按下状态 */
QPushButton:pressed {
    background-color: rgb(100, 100, 100); /* 按下时的背景色 */
    color: white; /* 按下时的文本颜色 */
}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="2" column="0">
        <widget class="QToolButton" name="btn_chart">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>生成图表</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/mainwindow/image/chart.png</normaloff>:/mainwindow/image/chart.png</iconset>
         </property>
         <property name="autoRepeatInterval">
          <number>100</number>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QToolButton" name="pbtn_incomeControl">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>收入管理</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/mainwindow/image/income.png</normaloff>:/mainwindow/image/income.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="pbtn_expenditure">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>支出管理</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/mainwindow/image/expenditure.png</normaloff>:/mainwindow/image/expenditure.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>30</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>10000</width>
        <height>10000</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Page</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Page</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actSelPopMenu">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/mainwindow/image/toolbox.png</normaloff>:/mainwindow/image/toolbox.png</iconset>
   </property>
   <property name="text">
    <string>工具箱</string>
   </property>
   <property name="toolTip">
    <string>工具箱</string>
   </property>
  </action>
  <action name="actCalculate">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/mainwindow/image/calculator.png</normaloff>:/mainwindow/image/calculator.png</iconset>
   </property>
   <property name="text">
    <string>计算器</string>
   </property>
   <property name="toolTip">
    <string>计算器</string>
   </property>
  </action>
  <action name="actExchangeRate">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/mainwindow/image/exchangeRate.png</normaloff>:/mainwindow/image/exchangeRate.png</iconset>
   </property>
   <property name="text">
    <string>汇率转换</string>
   </property>
   <property name="toolTip">
    <string>汇率转换</string>
   </property>
  </action>
  <action name="actUser">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/loginIMG/image/login.png</normaloff>:/loginIMG/image/login.png</iconset>
   </property>
   <property name="text">
    <string>用户</string>
   </property>
   <property name="toolTip">
    <string>用户信息</string>
   </property>
  </action>
  <action name="actUserChange">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/mainwindow/image/exchangeRate.png</normaloff>:/mainwindow/image/exchangeRate.png</iconset>
   </property>
   <property name="text">
    <string>切换账号</string>
   </property>
   <property name="toolTip">
    <string>切换账号</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/mainwindow/image/exchangeRate.png</normaloff>:/mainwindow/image/exchangeRate.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actUserInfo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/mainwindow/image/exchangeRate.png</normaloff>:/mainwindow/image/exchangeRate.png</iconset>
   </property>
   <property name="text">
    <string>用户信息</string>
   </property>
   <property name="toolTip">
    <string>用户信息</string>
   </property>
  </action>
  <action name="actTransltor">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/mainwindow/image/translator.png</normaloff>:/mainwindow/image/translator.png</iconset>
   </property>
   <property name="text">
    <string>翻译</string>
   </property>
   <property name="toolTip">
    <string>翻译</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
